Makefile.am

```diff
@@ -3,7 +3,7 @@ DIST_SUBDIRS = xdg
 
 BINDIR = $(srcdir)/bin
 
-PINTA_BUILD_OPTS = --configuration Release -p:BuildTranslations=true
+PINTA_BUILD_OPTS = --configuration Release -p:BuildTranslations=true --no-restore
```

Comment:
This breaks a normal build of Pinta using the makefile

Reply:
I'll take a look on the CI